com.microsoft.mrtk.graphicstools.unity/Samples~/Magnifier.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
Can you move this folder into the Experimental folder with Acrylic and CanvasShadow? (ie Samples~/Experimental/Magnifier)